### PR TITLE
SNO-143-switch-update-readmes

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,338 +1,58 @@
...
-  
-  
-- **JavaScript**: Make a syntax error, or style error, in a JS file. Warnings and errors should show on the left side of the line.  
-
+ENCODE Metadata Database
+========================
+
+
+## System Installation (OSX Catlina 10.15.2)
+See [Snovault OSX System Installation][].  Snovault is a dependecy of the Encoded application.
```
Comment (zoldello) [inline README.md]: type: "dependency" is incorrectly spelt "dependecy"

Comment (zoldello) [inline README.md]: Suggested wording: Encode installs Snovault as it is a dependency. However, you do not need to set up a running instance yourself.

File: README.md [hunk 2]
```diff
@@ -1,338 +1,58 @@
...
+Installing Snovault is not a requirement to run the this application, however, the
+system installation is the same.
+
+
+## Application Installation
+For issues see [Snovault OSX App Installation][] first.
+
+1. Create a virtual env in your work directory.
+    Here we use python3 venv module.  Use venv, like conda, if you please
```
Comment (zoldello) [inline README.md]: Suggested word: This example uses venv. Other options would also work like conda

File: README.md [hunk 3]
```diff
@@ -1,338 +1,58 @@
...
+    ```
+    $ cd your-work-dir
+    $ python3 -m venv encoded-venv
+    $ source encoded-venv/bin/activate
+    ```
+
+1. Clone the repo and install requirements
+    ```
+    $ cd encoded
+    # Make sure you are in the encoded-venv
```
Comment (zoldello) [inline README.md:24]: I think "# Make sure you are in the encoded-venv" should be mentioned once early. I believe saying it multiple times is noise.

File: README.md [hunk 4]
```diff
@@ -1,338 +1,58 @@
...
+
+
+## Application Installation
+For issues see [Snovault OSX App Installation][] first.
+
+1. Create a virtual env in your work directory.
+    Here we use python3 venv module.  Use venv, like conda, if you please
+    ```
+    $ cd your-work-dir
+    $ python3 -m venv encoded-venv
```
Comment (zoldello) [inline README.md:18]: In theory, no one should be using Python2 after January 1st, 2020... So, it "Python3" still needed?

File: README.md.pre-ubuntu18 [hunk 5]
```diff
@@ -0,0 +1,338 @@
+# ENCODE Metadata Database
```
Comment (zoldello) [inline README.md.pre-ubuntu18:1]: Are we still supporting ENCODE/SNOVAULT pre-ubuntu18? If no, do we need this page?

File: README.md.pre-ubuntu18 [hunk 6]
```diff
@@ -0,0 +1,338 @@
...
+-------------------------------------------------------
+
+
+[![CircleCI](https://circleci.com/gh/ENCODE-DCC/encoded/tree/dev.svg?style=svg)](https://circleci.com/gh/ENCODE-DCC/encoded/tree/dev)
+
+## Setting Up Your Environment
+
+These are the primary software versions used in production, and you should be able to use them locally:
```
Comment (zoldello) [inline README.md.pre-ubuntu18:9]: I do not see the point of mentioning this in that these Encode will not work without even one of the other dependencies.

File: README.osx.catalina.md [hunk 7]
```diff
@@ -1,58 +0,0 @@
-ENCODE Metadata Database
```
Comment (zoldello) [inline README.osx.catalina.md:1]: Why was this deleted?

Comment (zoldello) [reply README.osx.catalina.md:1]: Are we still supporting mojave and other older versions of mac?

**Review by zoldello — DISMISSED** [review]
Most of my comments are suggestions. There is one small typo that may need to be fixed (if you deem a wording suggestion irrelevant.)

File: README.osx.catalina.md (hunk 7 above)
Comment (caseylitton) [reply README.osx.catalina.md:1]: This is the README now.  Also having catalina in the file name was confusing for people on mojave.

File: README.md.pre-ubuntu18 (hunk 6 above)
Comment (caseylitton) [reply README.md.pre-ubuntu18:9]: These were the original instructions.  I didn't edit it, just moved the file.

File: README.md.pre-ubuntu18 (hunk 5 above)
Comment (caseylitton) [reply README.md.pre-ubuntu18:1]: I was going to keep it for a bit.  There are few external users of the repo who may still be on ubuntu14.  Although we could very well be the last people to use it!

File: README.md (hunk 4 above)
Comment (caseylitton) [reply README.md:18]: One day maybe!  The default python command on osx and ubuntu 18 is still python2.  

When I see 'python' in readme I assume python3, but don't mind the clarity when people write the 3.

**Review by hitz — APPROVED** [review]
